Libs/MRML/Core/vtkMRMLSliceNode.cxx

```diff
@@ -339,25 +379,22 @@ void vtkMRMLSliceNode::SetOrientation(const char* orientation)
     {
     return;
     }
- if (!strcmp(orientation, "Axial"))
-    {
-    this->SetOrientationToAxial();
-    }
-  else if (!strcmp(orientation, "Sagittal"))
-    {
-    this->SetOrientationToSagittal();
-    }
-  else if (!strcmp(orientation, "Coronal"))
-    {
-    this->SetOrientationToCoronal();
-    }
-  else if (!strcmp(orientation, "Reformat"))
-    {
-    this->SetOrientationToReformat();
-    }
-  else
+
+  if (!strcmp(orientation, "Reformat"))
```

Comment:
No need to check every time if an input is "Reformat". Just make sure "Reformat" string cannot be added as an orientation preset name and everywhere else you can rely on that.